src/web/app/pages-student/student-course-details-page/__snapshots__/student-course-details-page.component.spec.ts.snap

```diff
@@ -0,0 +1,501 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`StudentCourseDetailsPageComponent should snap with default fields 1`] = `
+<tm-student-course-details-page
+  Gender={[Function Object]}
+  backendUrl={[Function String]}
+  httpRequestService={[Function HttpRequestService]}
+  route={[Function ActivatedRoute]}
+  statusMessageService={[Function StatusMessageService]}
+  user="undefined"
+>
+  
+</tm-student-course-details-page>
+`;
+
+exports[`StudentCourseDetailsPageComponent should snap with populated fields 1`] = `
+<tm-student-course-details-page
+  Gender={[Function Object]}
+  backendUrl={[Function String]}
+  course={[Function Object]}
+  httpRequestService={[Function HttpRequestService]}
+  instructorDetails={[Function Array]}
+  route={[Function ActivatedRoute]}
+  statusMessageService={[Function StatusMessageService]}
+  student={[Function Object]}
+  teammateProfiles={[Function Array]}
+  user="undefined"
+>
+  <h1
+    class="page-title"
+  >
+     Thematic Systems
+  </h1><div
+    class="section"
+  >
+    <div
+      class="col-sm-12"
+    >
+      <h4
+        class="font-weight-bold"
+      >
+        Course
+      </h4>
+    </div>
+    <div
+      class="card border-primary"
+    >
+      <div
+        class="card-body fill-plain"
+      >
+        <div
+          class="form form-horizontal"
+        >
+          <div
+            class="form-group row"
+          >
+            <div
+              class="col-4 text-right"
+            >
+              <label>
+                Name
+              </label>
+            </div>
+            <div
+              class="col-8"
+            >
+              <span>
+                Thematic Systems
+              </span>
+            </div>
+          </div>
+          <div
+            class="form-group row"
+          >
+            <div
+              class="col-4 text-right"
+            >
+              <label>
+                ID
+              </label>
+            </div>
+            <div
+              class="col-8"
+            >
+              <span>
+                CS3281
+              </span>
+            </div>
+          </div>
+          <div
+            class="form-group row"
+          >
+            <div
+              class="col-4 text-right"
+            >
+              <label>
+                Timezone
+              </label>
+            </div>
+            <div
+              class="col-8"
+            >
+              <span>
+                Asia/Singapore
+              </span>
+            </div>
+          </div>
+          <div
+            class="form-group row"
+          >
+            <div
+              class="col-4 text-right"
+            >
+              <label>
+                Created at
+              </label>
+            </div>
+            <div
+              class="col-8"
+            >
+              <span>
+                1552509888215
```

Comment:
We can improve the presentation of the date here to make it look more readable to users (like in `sessions-table.component`)

Reply:
I think it's just printed as is received from the back end, so it should be fine as long the back end passes in something nice looking